setupbase.py

```diff
@@ -65,8 +65,27 @@ def execfile(fname, globs, locs=None):
 version_ns = {}
 execfile(pjoin(repo_root, name, '_version.py'), version_ns)
 
-version = version_ns['__version__']
```

Comment:
The changes in this file are no longer required, since `_version.py` is setting `__version__` to a normalized value.